Source/WebKit/Configurations/WebKit.xcconfig

```diff
@@ -266,7 +266,13 @@ WK_SPRINGBOARDSERVICES_LDFLAGS = $(WK_SPRINGBOARDSERVICES_LDFLAGS_$(WK_PLATFORM_
 WK_SPRINGBOARDSERVICES_LDFLAGS_iphoneos = -framework SpringBoardServices
 WK_SPRINGBOARDSERVICES_LDFLAGS_iphonesimulator = -framework SpringBoardServices;
 
-FRAMEWORK_AND_LIBRARY_LDFLAGS = -lobjc -framework CFNetwork -framework CoreAudio -framework CoreFoundation -framework CoreGraphics -framework CoreText -framework Foundation -framework ImageIO -framework IOKit -framework IOSurface -framework WebKitLegacy -lnetwork -framework Metal $(WK_APPSERVERSUPPORT_LDFLAGS) $(WK_ACCESSIBILITY_LDFLAGS) $(WK_APPKIT_LDFLAGS) $(WK_APPLE_PUSH_SERVICE_LDFLAGS) $(WK_BACKBOARD_SERVICES_LDFLAGS) $(WK_RUNNINGBOARD_SERVICES_LDFLAGS) $(WK_AUTHKIT_LDFLAGS) $(WK_CARBON_LDFLAGS) $(WK_CONTACTS_LDFLAGS) $(WK_CORE_PREDICTION_LDFLAGS) $(WK_CORE_SERVICES_LDFLAGS) $(WK_GRAPHICS_SERVICES_LDFLAGS) $(WK_LIBNETWORKEXTENSION_LDFLAGS) $(WK_LIBSANDBOX_LDFLAGS) $(WK_LIBWEBRTC_LDFLAGS) $(WK_MOBILE_CORE_SERVICES_LDFLAGS) $(WK_MOBILE_GESTALT_LDFLAGS) $(WK_MOBILE_KEY_BAG_LDFLAGS) $(WK_PDFKIT_LDFLAGS) $(WK_PROTOTYPE_TOOLS_LDFLAGS) $(WK_SAFE_BROWSING_LDFLAGS) $(WK_SECURITY_INTERFACE_LDFLAGS) $(WK_UIKIT_LDFLAGS) $(WK_MRUIKIT_LDFLAGS) $(WK_UNIFORM_TYPE_IDENTIFIERS_LDFLAGS) $(WK_URL_FORMATTING_LDFLAGS) $(WK_WEBINSPECTORUI_LDFLAGS) $(WK_COORDINATOR_LDFLAGS) $(WK_INSTALL_COORDINATION_LDFLAGS) $(WK_APP_STORE_DAEMON_LDFLAGS) $(WK_POWERLOG_LDFLAGS) $(WK_FRONTBOARDSERVICES_LDFLAGS) $(WK_LIBBSM_LDFLAGS) $(WK_NETWORK_EXTENSION_LDFLAGS) $(WK_BROWSERENGINEKIT_LDFLAGS) $(WK_WRITING_TOOLS_LDFLAGS) $(WK_USERNOTIFICATIONS_LDFLAGS) $(WK_UIKITSERVICES_LDFLAGS) $(WK_SPRINGBOARDSERVICES_LDFLAGS)
+WK_SCREENTIME_LDFLAGS = $(WK_SCREENTIME_LDFLAGS_$(WK_PLATFORM_NAME));
+WK_SCREENTIME_LDFLAGS_iphoneos = -framework ScreenTime;
+WK_SCREENTIME_LDFLAGS_iphonesimulator = -framework ScreenTime;
+WK_SCREENTIME_LDFLAGS_macosx = -framework ScreenTime;
```

Comment:
I still wonder if this is actually available in all kinds of macOS and iOS builds, or if this is going to break some obscure build. Richard (or any of us, but especially Richard) can help with commands to confirm :)

Reply:
indeed. I'll write up a paragraph or so explaining all of this in a bit.